libsolidity/analysis/TypeChecker.cpp

```diff
@@ -387,7 +387,7 @@ bool TypeChecker::visit(FunctionDefinition const& _function)
 			_var.referenceLocation() == VariableDeclaration::Location::Storage &&
 			!m_currentContract->abstract()
 		)
-			m_errorReporter.typeError(
+			m_errorReporter.fatalTypeError(
```

Comment:
This generally seems fine to me but please check if it shouldn't rather be fixed in the same place as #11000 instead.